vision/body_analysis/arcface/dependencies/verification.py

```diff
@@ -1,27 +1,7 @@
-"""Helper for evaluation on the Labeled Faces in the Wild dataset 
-"""
+# SPDX-License-Identifier: Apache-2.0
 
-# MIT License
```

Comment:
we cannot remove the license for someone else's file

Reply:
In that case, it will be tricky since in some other ONNX projects, it was still in MIT license when the code was contributed.
@prasanthpul